ivy/func_wrapper.py

```diff
@@ -962,34 +962,44 @@ def _wrap_function(
         # set attributes
         for attr in original.__dict__.keys():
             # private attribute or decorator
-            if attr.startswith("_") or hasattr(ivy, attr) or attr == "handles_out_arg":
+            if (
+                attr.startswith("_")
+                or hasattr(ivy, attr)
+                or (not compositional and attr == "mixed_function")
```

Comment:
given that we won't have the `mixed_function` attribute, let's remove this check

Reply:
Oops, had missed it. Have removed it now.